pom.xml

```diff
@@ -1087,6 +1087,18 @@
                                 </configuration>
                             </execution>
 
+                            <execution>
+                                <phase>prepare-package</phase>
```

Comment:
Why do we hook up into `prepare-package` now'?

Reply:
it's to avoid doing this during normal build to speed up development

Reply:
Would that not mean that if you have in `blueocean-core-js/package.json` 

```
+    "mvnbuild": "gulp bundle",
     "mvntest": "gulp test",
+    "mvnpackage": "gulp build",
```

We now bundle twice?

Reply:
notice one is `build` and the other is `bundle`, it's a stop-gap to make the code available for tools without publishing to npm (we can change this to be more efficient later, but this helps the current dev case, at least)